crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/validation/CrownCourtDetailsValidatorTest.java

```diff
@@ -0,0 +1,104 @@
+                .isInstanceOf(ValidationException.class)
+                .hasMessageContaining(CrownCourtDetailsValidator.MSG_INVALID_CC_OUTCOME);
+    }
+
+    @Test
+    public void givenLatestCrownCourtOutcomeConvicted_whenValidateIsInvoked_thenValidationPasses() {
+        ApiCrownCourtOutcome crownCourtOutcome1 = TestModelDataBuilder.getApiCrownCourtOutcome(CrownCourtOutcome.AQUITTED, LocalDateTime.now().minusDays(2));
+        ApiCrownCourtOutcome crownCourtOutcome2 = TestModelDataBuilder.getApiCrownCourtOutcome(CrownCourtOutcome.SUCCESSFUL, LocalDateTime.now().minusDays(1));
+        ApiCrownCourtOutcome crownCourtOutcome3 = TestModelDataBuilder.getApiCrownCourtOutcome(CrownCourtOutcome.CONVICTED, null);
+        ApiCrownCourtSummary crownCourtSummary = TestModelDataBuilder.getCrownCourtSummaryWithOutcome(false, Arrays.asList(crownCourtOutcome1, crownCourtOutcome2, crownCourtOutcome3));
+
+        assertThat(crownCourtDetailsValidator.validate(crownCourtSummary)).isEmpty();
+    }
+
+    @Test
+    public void givenLatestCrownCourtOutcomeSuccessfulWithNullDate_whenValidateIsInvoked_thenValidationPasses() {
+        ApiCrownCourtOutcome crownCourtOutcome1 = TestModelDataBuilder.getApiCrownCourtOutcome(CrownCourtOutcome.AQUITTED, LocalDateTime.now().minusDays(2));
+        ApiCrownCourtOutcome crownCourtOutcome2 = TestModelDataBuilder.getApiCrownCourtOutcome(CrownCourtOutcome.CONVICTED, LocalDateTime.now().minusDays(1));
+        ApiCrownCourtOutcome crownCourtOutcome3 = TestModelDataBuilder.getApiCrownCourtOutcome(CrownCourtOutcome.SUCCESSFUL, null);
+        ApiCrownCourtSummary crownCourtSummary = TestModelDataBuilder.getCrownCourtSummaryWithOutcome(null, Arrays.asList(crownCourtOutcome1, crownCourtOutcome2, crownCourtOutcome3));
+
+        assertThat(crownCourtDetailsValidator.validate(crownCourtSummary)).isEmpty();
+    }
+
+    @Test
+    public void givenValidCrownCourtOutcomeSuccessful_whenValidateIsInvoked_thenValidationPasses() {
+        ApiCrownCourtOutcome crownCourtOutcome = TestModelDataBuilder.getApiCrownCourtOutcome(CrownCourtOutcome.SUCCESSFUL, LocalDateTime.now());
+        ApiCrownCourtSummary crownCourtSummary = TestModelDataBuilder.getCrownCourtSummaryWithOutcome(false, Arrays.asList(crownCourtOutcome));
+
+        assertThat(crownCourtDetailsValidator.validate(crownCourtSummary)).isEmpty();
+    }
+
+    @Test
+    public void givenValidCrownCourtOutcomeWithNullDate_whenValidateIsInvoked_thenValidationPasses() {
+        ApiCrownCourtOutcome crownCourtOutcome = TestModelDataBuilder.getApiCrownCourtOutcome(CrownCourtOutcome.SUCCESSFUL, null);
+        ApiCrownCourtSummary crownCourtSummary = TestModelDataBuilder.getCrownCourtSummaryWithOutcome(null, Arrays.asList(crownCourtOutcome));
+
+        assertThat(crownCourtDetailsValidator.validate(crownCourtSummary)).isEmpty();
+    }
+
+    @Test
+    public void givenCrownCourtOutcomeConvictedWithNullDateAndNullImprisoned_whenValidateIsInvoked_thenValidationFails() {
```

Comment:
please remove the public modifier from the test methods

Reply:
an oversight. I'll remove these